packages/terra-layout/src/LayoutSlidePanel.module.scss

```diff
@@ -59,6 +59,7 @@
 
     > .panel {
       border-right: 0.07143rem solid var(--layout-slide-panel-panel-border-color, #d3d4d5);
+      border-right-width: var(--terra-layout-slide-panel-squish-panel-border-right-width, 1px);
```

Comment:
This variable is pulled from the [OCS theme branch](https://github.com/cerner/terra-framework/blob/3152c95dd5c9c39f028a92d2d2e884549d282331/packages/terra-layout/src/LayoutSlidePanel.module.scss#L72)